### DISCUSSION TITLE
Possible issue with Actions

### DISCUSSION THREAD

**Discussion by thecodeassassin** [community/community#77211] [Actions] [answered]
### Select Topic Area

Question

### Body

Is there currently an issue with GIthub actions? I checked the status page and all is green there but none of our builds are being triggered after a push.

EDIT: Seems GH took notice and resolved the issue

**Comment by SoftwarehouseForAutomation** [comment]
faced the same issue, no action is triggered

**Comment by sean-dickinson** [comment]
Seeing the same behavior

**Comment by circlecube** [comment]
Seeing the same issue.

**Comment by zetas** [comment]
same behavior. Also had issues running the checkout@v3 action

**Comment by jcrides** [comment]
Seeing the same thing. No workflows are being triggered.

**Comment by MateusTonelli** [comment]
Seeing the same issue.

**Comment by ScentAndCo** [comment]
Same issue here, I just posted a question about it, glad its not just me

**Comment by srelf-eschercloud** [comment]
Also seeing this issue, some work flows not being triggered.

on_push looks to be broken.

**Comment by matteos025** [comment]
Same issue here

**Comment by jmtos** [comment]
I've been trying for more than 15 minutes, Github needs to inform us about this occurrence

**Comment by undeologist** [comment]
"Is there currently an issue with GIthub actions? I checked the status page and all is green there but none of our builds are being triggered after a push." 

We are facing the same problem .

**Comment by lucas1619** [comment]
Same issue, no actions is triggered

**Comment by ZeroHans4** [comment]
Same issue here

**Comment by EnzeGH** [comment]
Same issue

**Comment by chpwssn** [comment]
Same issue here for branch pushes but it's looking like `tag` actions are triggering ok.

**Comment by sriksunny** [comment]
Facing the same issue - no action firing on any of our repos

**Comment by lynicis** [comment]
oh i'm not alone :)

**Comment by naresh-gn** [comment]
same issue here

**Comment by mpechner-akasa** [comment]
yup, same

**Comment by mackenly** [comment]
It seems to work when you manually trigger a workflow.

**Reply by naresh-gn** [reply]
yup, workflow_dispatch and manually trigger previous run is working but commit push is not working

**Comment by jmtos** [comment]
the service has apparently been reestablished for me

**Reply by ScentAndCo** [reply]
yep mine seems to be working again as well

**Comment by lynicis** [comment]
it works

**Comment by davidbudnick** [comment]
Looks as GH is looking into the issue: https://www.githubstatus.com/incidents/dsb3kn1zfdl7

**Reply by davidbudnick** [reply]
But it does work for me atm 🤷🏻‍♂️

**Comment by sg-sbhardwa** [comment]
I guess its working now, atleast my workflow just worked on push. FYI this issue was only for self hosted runners not with the github runners.

**Comment by rosko** [comment, marked as answer]
It was the same issue for me. Now, it's working. Using standard/default runner.

**Reply by aqsaqayyum** [reply]
Hi Rosko, Did you have idea how can we push updated code in already clone repo? It's always giving close option know.

**Comment by phrfpeixoto** [comment]
I'm still having isssues. Cloning the repo is taking a lifetime!
<img width="1305" alt="Screenshot 2023-11-28 at 20 45 26" src="https://github.com/community/community/assets/10242829/5ba735ec-dbd5-4881-a38d-44fd8ac47402">

**Comment by benjefferies** [comment]
I'm experiencing the same thing on tag events and pushing commits
